src/commands/linkAdmin.ts

```diff
@@ -0,0 +1,128 @@
+import { logger } from "../logger";
+import {
+  inlineCode,
+  SlashCommandBuilder,
+  SlashCommandStringOption,
+  SlashCommandSubcommandBuilder,
+} from "@discordjs/builders";
+import { CommandInteraction, CacheType, AutocompleteInteraction, MessageActionRow, MessageSelectMenu } from "discord.js";
+import { CommandType } from "../types";
+import fs from "fs";
+import { Link, linkModel } from "../schemas/link";
+import path from "path";
+import { CreateNewLink, FindLinkByName, GetAllShortenLinks } from "../helpers/linkQueries";
+
+
+const linkAdminModule: CommandType = {
+  data: new SlashCommandBuilder()
+    .setName("link-admin")
+    .setDescription("Which link?")
+    .addSubcommand(subcommand =>
+      subcommand
+        .setName("create")
+        .setDescription("Add new link?")
```

Comment:
A command description is less a question and more a statement of what the user can achieve with the command. For example "Add a new link" or "Delete an existing link"